tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -321,6 +321,15 @@ generate_qemu_options() {
 	# Don't build the qemu-io, qemu-nbd and qemu-image tools
 	qemu_options+=(size:--disable-tools)
 
+	# Kata Containers may be configured to use the virtiofs daemon.
+	#
+	# But since QEMU 5.2 the daemon is built as part of the tools set
+	# (disabled with --disable-tools) thus it needs to be explicitely
+	# enabled.
+	if [[ "${qemu_version_major}" -gt 5 || ("${qemu_version_major}" -eq 5 && "${qemu_version_minor}" -ge 2) ]]; then
```

Comment:
Please use `bc` for these sorts of version comparision. We see bugs time and time again using this sort of logic. Here's an example (from this same script):

https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/scripts/configure-hypervisor.sh#L253